src/Files.App/ViewModels/SettingsViewModels/PreferencesViewModel.cs

```diff
@@ -112,6 +112,9 @@ public PreferencesViewModel()
 
 			AddDateTimeOptions();
 			SelectedDateTimeFormatIndex = (int)Enum.Parse(typeof(DateTimeFormats), DateTimeFormat.ToString());
+			SelectedDefaultSortingIndex = DefaultSortOption == SortOption.FileTag ? 5 : (int)DefaultSortOption;
```

Comment:
Where does the `5` comes from? Maybe you could put it inside a well named variable to avoid the magic constant effect?

Reply:
This is a workaround to hide "sync status" and other options which only work in cloud folders or recycle bin. I tried swapping sync status and tag in SortOption and GroupOption, but it should be done in toolbar and context menu too, otherwise the wrong option is selected there.

Reply:
If it's a work around, I again highly suggest using a named variable to explain its use. Or at least a comment next to it.